product_variant_available_in_pos/__manifest__.py

```diff
@@ -1,15 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Alex Comba - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Product Variant Available In Pos',
     'summary': 'Available In Pos in product level',
-    'version': '8.0.1.0.0',
+    'version': '11.0.1.0.0',
     'category': 'Point Of Sale',
     'license': 'AGPL-3',
     'author': 'Agile Business Group, Odoo Community Association (OCA)',
-    'website': 'https://www.agilebg.com/',
+    'website': 'https://github.com/OCA/product-variant/tree/'
```

Comment:
Please use repository URL, not full one

Reply:
I actually prefer the full link, so when someone clicks on it, they directly see the module's (brand new) README. 
Is there any side effect in using the full link?

Reply:
It forces you to change it each time you migrate the module. The README is displayed at the same time of the manifest, so I don't see this very useful.

Reply:
This link is not working anyway.

Reply:
@pedrobaeza the link should work when the PR is merged, it is the same as https://github.com/OCA/product-variant/tree/8.0/product_variant_available_in_pos
but just with 11.0 instead of 8.0

Reply:
OK, I don't get why do you prefer this long form, but I don't block more.